### PR TITLE
footer & mainAppComponent

### PR DESCRIPTION
**Pull request by OliviaWinteringham**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -22,7 +23,7 @@ export default class App extends Component {
   render() {
     return (
       <View style={styles.container}>
-        <FlatListBasics />
+        <MainAppComponent />
```
Comment (OliviaWinteringham) [inline App.js:26]: Refactor Flat list to another view later

File: components/FlatList/index.js [hunk 2]
```diff
@@ -3,27 +3,21 @@ import { FlatList, StyleSheet, Text, View } from "react-native";
...
-            <View>
-              <ListItem bottomDivider={true} title={item.poiName} />
-              <Text>{item.poiTrivia}</Text>
-            </View>
-          )}
-        />
-      </View>
-      <FooterSection />
+    <View style={styles.container}>
+      <FlatList
```
Comment (OliviaWinteringham) [inline components/FlatList/index.js:10]: Bob on. single use flatlist sorted

**Review by OliviaWinteringham — COMMENTED** [review]
looks good

**Review by Aardeedev — APPROVED** [review]
Sahela is awesome and also only 1 sahela high

**Pull request merged**